src/autocomplete.js

```diff
@@ -356,8 +363,11 @@ yourlabs.Autocomplete.prototype.initialize = function() {
      */
     this.box
         .on('mouseenter', this.choiceSelector, $.proxy(this.boxMouseenter, this))
-        .on('mouseleave', this.choiceSelector, $.proxy(this.boxMouseleave, this))
+        .on('mouseleave', this.choiceSelector, $.proxy(this.boxMouseleave, this));
+    if(this.bindMouseDown){
+        this.box
         .on('mousedown', this.choiceSelector, $.proxy(this.boxClick, this));
+    }
```

Comment:
I'm affraid this is going to be too complicated if we start adding a variable for every possible behaviour.

Would it be possible to add a new method instead, to encapsulate the thix.box.on calls, and allow the user to override this method in their own project ?